core/unit_test/TestTimer.hpp

```diff
@@ -0,0 +1,53 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include <gtest/gtest.h>
+
+#include <Kokkos_Timer.hpp>
+
+#include <chrono>
```

Comment:
This should be already included in `Kokkos_Timer.hpp`. Remove it from here to ensure `Kokkos_Timer.hpp` is self-standing.

Reply:
We have self containment tests for headers.
That said I can remove that header include if you feel strongly about it.

Reply:
Please.

Reply:
```suggestion
```